reference/entry.json

```diff
@@ -2006,6 +2006,259 @@
           "Places"
         ],
         "x-internal": false
+      },
+      "put": {
+        "summary": "",
```

Comment:
Remember to add a summary to give this operation a human-readable link in the sidebar